src/main/java/org/folio/innreach/domain/service/impl/CirculationServiceImpl.java

```diff
@@ -499,6 +508,48 @@ private void updateTransactionHold(TransactionHold existingTransactionHold, Tran
     BeanUtils.copyProperties(pickupLocation, existingTransactionHold.getPickupLocation(), PICKUP_LOC_IGNORE_PROPS_ON_COPY);
   }
 
+  private void reportVerifyPatron(InnReachTransaction transaction) {
+    var payload = new HashMap<>();
+    var patronHold =  (TransactionPatronHold) transaction.getHold();
+    var visiblePatronId = patronHold.getPatronId();
+    var patronAgencyCode = patronHold.getPatronAgencyCode();
+    var patronName = patronHold.getPatronName();
+    payload.put("visiblePatronId", visiblePatronId);
+    payload.put("patronAgencyCode", patronAgencyCode);
+    payload.put("patronName", patronName);
+    callD2irCircOperation(D2IR_VERIFY_PATRON, transaction, payload);
+  }
```

Comment:
this call makes no sense, there is no such endpoint on central server side. This is an endpoint that we implement. Take a look at `PatronInfoController`